### PR TITLE
test(backend): add tests for `ThresholdUtils`

### PR DESCRIPTION
**Pull request by Ragnhilmd**

### CODE REVIEW

File: backend/tests/Backend.Tests/UnitTests/Utils/ThresholdsUtilsTests.cs [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+using Backend.Utils;
+
+namespace Backend.Tests.UnitTests.Utils;
+
+/// <summary>
+/// Unit tests for ThresholdUtils to verify danger level calculation logic.
+/// </summary>
+/// <remarks>
+/// These tests cover the public helper methods used to evaluate threshold states for noise, dust, and vibration data.
+/// They focus on threshold boundaries, peak evaluation, field-specific overrides, and the cumulative day-based vibration flow.
```
Comment (WilliamTjonndal) [inline backend/tests/Backend.Tests/UnitTests/Utils/ThresholdsUtilsTests.cs]: What are the thresholds boundaries? The documentation/comments should mention the exact limits (numbers) of what we expect something to be "safe", "warning" or "danger". You have already used these values, but they look like magic number right now, like at line 44, what do these numbers represent? Maybe you can create constants like so below:
```cs
const int NOISE_WARNING_THRESHOLD = 80;
const int NOISE_DANGER_THRESHOLD = 85;
const int NOISE_PEAK_DANGER = 130;```

**Comment by WilliamTjonndal** [comment]
It could also be an idea to test for extreme values, like negative values, malformed data, or no user logged in (user id is null), but this may not be necessary.

**Review by andrefkl — APPROVED** [review]
